src/main/fc/fc_tasks.c

```diff
@@ -103,20 +106,37 @@
 #endif
 
 #ifdef USE_BST
-void taskBstMasterProcess(timeUs_t currentTimeUs);
+static void taskBstMasterProcess(timeUs_t currentTimeUs);
```

Comment:
`static` on a function declaration makes no sense, it's defined elsewhere anyway, in `i2c_bst.c`